src/views/admin/organisationen/SchuleCreationView.vue

```diff
@@ -346,22 +349,22 @@
             >
               {{ $t('admin.schule.schulform') }}:
             </v-col>
-            <v-col class="text-body"
-              ><span data-testid="created-schule-form">{{ preservedSchulform }}</span></v-col
-            >
+            <v-col class="text-body">
+              <span data-testid="created-schule-form"> {{ preservedSchulform }}</span>
+            </v-col>
           </v-row>
           <v-row>
             <v-col
               class="text-body bold text-right"
-              data-testid="created-schule-dienststellennummer-label"
+              data-testid="created-dienststellennummer-label"
```

Comment:
TestId should not change, should it?
```suggestion
              data-testid="created-schule-dienststellennummer-label"
```